chain/client/src/sync/state.rs

```diff
@@ -319,24 +317,10 @@ impl StateSync {
                     )?;
                 }
                 ShardSyncStatus::ReshardingScheduling => {
-                    debug_assert!(need_to_reshard);
-                    self.sync_shards_resharding_scheduling_status(
-                        shard_id,
-                        shard_sync_download,
-                        sync_hash,
-                        chain,
-                        resharding_scheduler,
-                        me,
-                    )?;
+                    panic!("Resharding V2 scheduling is no longer supported")
```

Comment:
Are you planning to follow up and delete those entirely?

Reply:
Indeed, if I get to delete `ShardSyncStatus::ReshardingScheduling` this part will be gone as well